### DISCUSSION TITLE
GithubBOX to CodeSandBox, is it an Official spec?

### DISCUSSION THREAD

**Discussion by muratumutlu** [codesandbox/codesandbox-client#3998] [General] [answered]
Hello to all,

A few days ago I have seen an easy way of using Github with Codesandbox for Front End projects.

I wonder that is it an official method?

Sample:
[](https://github.com/muratumutlu/cv-resume-react-nextjs-tailwindcss)
[](https://githubBOX.com/muratumutlu/cv-resume-react-nextjs-tailwindcss)

**Comment by garethx** [comment, marked as answer]
Hey - no, it's not official it was created by a community member. The functionality exists natively (see https://codesandbox.io/docs/importing#import-from-github), but it adds a handy domain to it.
